packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -150,19 +150,17 @@ class _OpenUpwardsPageTransition extends StatelessWidget {
 
 // Zooms and fades a new page in, zooming out the previous page. This transition
 // is designed to match the Android 10 activity transition.
-class _ZoomPageTransition extends StatefulWidget {
+class _ZoomPageTransition extends StatelessWidget {
+  /// The [animation] and [secondaryAnimation] argument are required and must
```

Comment:
the docs should also include a description of the constructor (or you can change `///` to `//` to indicate it's not camera-ready documentaion)

Reply:
```dart
/// Creates a [_ZoomPageTransition].
///
/// The [animation] and [secondaryAnimation] argument are required and must
/// not be null.
```

Something like this?

Reply:
Or we can remove these comments since the transition is a hidden one.